docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -24,12 +24,16 @@
     {{ end }}
 {{ end }}
 
-{{ $section := .CurrentSection }}
-{{ $api_section := .Site.GetPage "/docs/apis/_index.md" }}
-{{ $in_api_section := (.IsDescendant $api_section) }}
+{{ $selectedApiVersion := ""}}
```

Comment:
As we discussed in the meeting yesterday, this should probably default to `currentVersion` from `versions.json`.

We can make sure we have the nice landing pages we need by copying what is currently in `docs/_replacements/docs/apis` into `docs/_replacements/docs/apis/1.0` and `docs/_replacements/docs/apis/2.0`.

Reply:
Do we turn on `1.0` only now? should we have `previousVersion` here?

Reply:
Awesome. Updated